### PR TITLE
Release 4.1.0

### PR DESCRIPTION
**Pull request by richaferry**

### CODE REVIEW

File: press-permit-core.php [hunk 1]
```diff
@@ -111,15 +112,15 @@ function () use ($min_wp_version) {
     if ($pro_active) {
         add_filter(
             'plugin_row_meta',
-            function($links, $file)
-            {
+            function ($links, $file) {
                 if ($file == plugin_basename(__FILE__)) {
-                    $links[]= __('<strong>This plugin can be deleted.</strong>', 'revisionary');
+                    $links[] = __('<strong>This plugin can be deleted.</strong>', 'revisionary');
```
Comment (agapetry) [inline press-permit-core.php:117]: @richaferry At some point, this text domain should be changed, but it's not crucial for this release.

Comment (richaferry) [reply press-permit-core.php:117]: I think this line changed coming from php formatter in my vscode @agapetry . I didn't changed that line intendeedly. Is it need i revert that line?

Comment (agapetry) [reply press-permit-core.php:117]: It should pass 'press-permit-core' instead of 'revisionary'

Comment (richaferry) [reply press-permit-core.php:117]: Ah i see. I got your point. nice catch @agapetry .

Comment (richaferry) [reply press-permit-core.php]: Already fix in this commit https://github.com/publishpress/PublishPress-Permissions/pull/1313/commits/b3c22186398ef7fd8bf642ab9b5ecd3bd37caad8 @agapetry 
Thank you kevin. :+1:

**Pull request merged**